requirements.txt

```diff
@@ -1,16 +1,25 @@
 alembic==1.4.2
+attrs==19.3.0
```

Comment:
We can drop this requirement now — I was using it before I found out about built-in `@dataclass`, but it's no longer used anywhere in the PR code and I don't expect to use it in the future.

Reply:
It's required by another package, it turns out.

Reply:
OHHHHH right ok. Good.

Reply:
(This also answers my general question about `requirements.txt` above.)

Reply:
Yeah, we should do a review soonish, and uninstall the ones we don't know about one at a time and see if things break.